pages/CLA.md

```diff
@@ -10,9 +10,9 @@ You accept and agree to the following terms and conditions for Your present and
 
 ### Licenses
 
-jQuery Foundation projects (code, documentation, and any other materials) are released under the terms of the [MIT license](http://en.wikipedia.org/wiki/MIT_License), with the following exception:
+jQuery Foundation projects (code, documentation, and any other materials) are each released under the terms of the individual licenses as noted in each project repository, or, if no separate license is specified, under the terms of the [MIT license](https://tldrlegal.com/license/mit-license), with the following exception:
```

Comment:
>"if no separate license is specified, under the terms of the MIT license"

>I'm not sure that's legally possible. If there is no license, there is no license. @stevenayr can the >CLA really assign a license to arbitrary projects?

The idea is obviously to always have a license file in each repo, and for that to be where people go to find out what the license is for each project they're contributing to. So by having this language in the CLA, and then having a corresponding license file, as a matter of evidence we could that a person has signed the CLA on X date, and that they contributed on Y date, and that on Y date, this is what the license file looked like, so that's what they're tied to. 

The purpose of this line then is just to operate as a catch-all or backstop in the event there's no license file in a repo, or there's some question about whether something is licensed and how. In other words, it's our default rule subject to change by individual license files.

As a legal matter, there's no particular significance to the license file in the repo, apart from the fact that it is a convenient and customary way of showing what the license is in a way that it would be difficult for a contributor to deny seeing. However, the whole license issue is just a matter of a contract between the contributor and the foundation, wherein the contributor agrees to license certain code under certain terms, and the foundation agrees to hold that code under certain terms. It doesn't really matter whether that contract, and the license terms, are expressed in the CLA or in the license file, or pasted on the front page of jquery.org, it's all the same agreement. From the foundation's perspective, we are most concerned with making clear what the license terms will be ahead of time so as to prevent any disputes and having a legally enforceable agreement in the event there is a dispute, both of which I think are accomplished here. 

As for "arbitrary projects", I don't think that the term "project" as it's used within the foundation really has any legal significance. That is, if there's code contributed to a foundation repo, whether within what we would typically think of as a "project" or not, then it's covered by the CLA, and if for whatever reason there's not an operative license file, then it's contributed under MIT.